3rdparty/voice_text/src/voice_text.cpp.in

```diff
@@ -21,7 +21,7 @@
 #ifdef USE_DUMMY_INCLUDE
 #include "dummy/vt_dummy.h"
 #else
-#include "@VT_ROOT@/inc/vt_jpn.h"
+#include "@VT_INCLUDE@"
```

Comment:
you can write
```
#include VT_INCLDUE
```
and compile this with `-DCV_INCLUDE`
so add 
```
```
to camke file, for example
```
add_definitions(-D${VT_INCLUDE})
```
or
```
  set_target_properties(voice_text PROPERTIES COMPILE_FLAGS -D${VT_INCLUDE})
```
or use
```
#include "vt_jpn.h"
```
and
```
include_directories(${VT_ROOT}/inc) ????
```